
Header header

# scan or scan_cloud is set
sensor_msgs/LaserScan scan
sensor_msgs/PointCloud2 scan_cloud

GlobalDescriptor global_descriptor